observability/dashboards/all/argocd.yaml

```diff
@@ -973,7 +973,7 @@ spec:
         {
           "datasource": {
             "type": "loki",
-            "uid": "logging-infrastructure"
```

Comment:
Bitte Dashboard genauer anschauen, wir haben keinen Zugriff auf die Infrastructure Logs -> Bitte nur unsere Argos Monitoren.

Reply:
ArgoCD ist mittlerweile in gepardec-run-gitops und nicht mehr in einem openshift-* Namespace, daher funktioniert das.